domain/schema/model/sql/0021-status.sql

```diff
@@ -1,24 +1,53 @@
--- Status values for machines
-CREATE TABLE machine_status_values (
+-- Status values for machines.
```

Comment:
I know this has already been created by why are we smashing all the statuses that are unrelated in the same file. The only thing they share in common is the word "status". Should they not live with the entities that they exist to serve and related with ?

Reply:
I have no idea. I had the same thought. This has been previously approved and I am sticking with convention. I plan on raising it and we'll need to fix if agreed.

Reply:
These lookup tables have been relocated to their respective domain packages